.github/workflows/tf-apply-hash.yml

```diff
@@ -0,0 +1,129 @@
+name: HASH Terraform Apply
+
+# TODO: While this workflow is specific to HASH, we should consider
+#  making this a reusable workflow, parameterized on the directory and environment.
+
+on:
+  workflow_dispatch:
+    inputs:
+      env:
+        type: choice
+        options: ["prod"]
+        description: "Environment to deploy"
+        required: true
+        default: "prod"
+
+  push:
+    branches:
+      - main
+    paths:
+      - "infra/internal-api/**"
+
+env:
+  VAULT_ADDR: ${{ secrets.VAULT_ADDR }}
+
+jobs:
+  matrix-eval:
+    name: Prepare environments to be applied
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - name: Set matrix
+        id: set-matrix
+        env:
+          ENV: ${{ github.event.inputs.env }}
+        run: |
+          # If no env is specified, run both dev and prod
```

Comment:
Either this comment needs correcting or the matrix below does

Reply:
I've removed the conditional as it is superfluous 3e654bb45e31fd64c490ea1fe30ff6ed005a3186